doc/tutorial/pairwise_alignment/pairwise_alignment_solution_4.cpp

```diff
@@ -15,7 +15,7 @@ int main()
     // Configure the output:
     auto output_config = seqan3::align_cfg::output_score |
                          seqan3::align_cfg::output_begin_position |
-                         seqan3::align_cfg::output_end_position |
+                         seqan3::align_cfg::output_end_position{} |
```

Comment:
I see that you adapted `output_begin_position` and the other configurations as well such that the interface becomes consistent again. 👍